README.md

```diff
@@ -32,16 +32,16 @@ You can view the generated [reference docs][javadoc] for a full list of classes
 ## Requirements
 
 * Android API level 15+
-* Maps SDK via Google Play Services (this library is not yet compatible with the [Maps SDK v3.0 BETA] library)
+* Maps SDK v2 via Google Play Services OR [Maps SDK v3 BETA] library
```

Comment:
We're technically not branding this as "V2" (even though it is) in our documentation so I think we should keep this as "Maps SDK via Google Play Services" to avoid confusion.

Reply:
Sounds good - I just updated this in https://github.com/googlemaps/android-maps-utils/pull/757/commits/62a1cb24393cba434594bf05df37a8900634d352.